src/components/ActivityForm.js

```diff
@@ -149,13 +150,11 @@ export default class ActivityForm extends React.Component {
           />
           <select value={day} onChange={this.onInputValueChange('day')} required>
             <option value="">Pick a day</option>
-            <option value={1}>Monday</option>
-            <option value={2}>Tuesday</option>
-            <option value={3}>Wednesday</option>
-            <option value={4}>Thursday</option>
-            <option value={5}>Friday</option>
-            <option value={6}>Saturday</option>
-            <option value={7}>Sunday</option>
+            {daysOfTheWeek.map(weekDay => (
+              <option value={weekDay} key={weekDay}>
+                {`${weekDay.charAt(0).toUpperCase()}${weekDay.slice(1)}`}
```

Comment:
`text-transform: capitalize;` would be better. It's a micro performance improvement, but you can unblock your thread (js is single threaded lang) and put this operation to css, so another engine will take care about this operations. 

You should start to think about separate `Dropdown` component. It's a hard thing to do, using native html, it's harder to style options, etc. You could build it by yourself, but also you can use on of the most popular react packages (atm) - `rc-dropdown` or `downshift`. 
It will be big exercise. So, do it later (when you start real styling - do it also quite asap). But by separate pr only for dropdown.

Reply:
added to "TODO comments" in code